Spanner/src/Connection/Grpc.php

```diff
@@ -679,6 +681,22 @@ public function createDatabase(array $args)
         return $this->operationToArray($res, $this->serializer, $this->lroResponseMappers);
     }
 
+    /**
+     * @param array $args
+     */
+    public function updateDatabase(array $args)
+    {
+        $database = $this->pluck('database', $args);
+        $databaseInfo = $this->serializer->decodeMessage(new Database(), $database);
+        $databaseName = $databaseInfo->getName();
+        $updateMask = $this->serializer->decodeMessage(new FieldMask(), $this->pluck('updateMask', $args));
```

Comment:
I think it would look nice if we either followed this approach of using the `$this->pluck()` directly or the above approach(line 689) of first storing the value in a variable then using it.

Reply:
makes snese